packages/react-divider/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@fluentui/react-divider",
+  "version": "0.1.0",
+  "description": "React components for building web experiences",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/react-conformance": "^1.0.0",
+    "@fluentui/scripts": "^1.0.0",
+    "@testing-library/react": "^10.4.9",
+    "@testing-library/react-hooks": "^5.0.3",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
```

Comment:
Let's try to remove all enzyme deps if possible and focus on `@testing-library/react`

Reply:
Lets have a separate PR to move all converged components to testing-library/react